src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,63 @@
+package mate.academy.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
```

Comment:
lets add toString method(without salt and password fields) to display user correct in exception messages